includes/payments/class-wc-woomercadopago-basic-gateway.php

```diff
@@ -50,6 +50,7 @@ public function __construct() {
 		parent::__construct();
 		$this->form_fields         = $this->get_form_mp_fields( 'Basic' );
 		$this->hook                = new WC_WooMercadoPago_Hook_Basic( $this );
+		$this->notification        = new WC_WooMercadoPago_Notification_Webhook( $this );
```

Comment:
Para o basic_gateway, tem que utilizar new WC_WooMercadoPago_Notification_Ipn( $this )